packages/pancake-uikit/src/widgets/Menu/icons/Farm.tsx

```diff
@@ -1,24 +0,0 @@
-import React from "react";
-import Svg from "../../../components/Svg/Svg";
-import { SvgProps } from "../../../components/Svg/types";
-
-const Icon: React.FC<SvgProps> = (props) => {
```

Comment:
Awesome to delete all these - can you check they aren't used in the FE somewhere though - some find there way onto one-off cards?

If they are used - move them over to where the regular icons live 👍